package.json

```diff
@@ -20,8 +20,8 @@
     },
     "dependencies": {
         "@types/uuid": "^8.3.4",
-        "cosmian_cover_crypt": "^6.0.8",
-        "cosmian_findex": "^0.9.0",
```

Comment:
To change after https://github.com/Cosmian/cover_crypt/pull/55 and http://gitlab.cosmian.com/core/findex/-/merge_requests/41